docs/SetupBase9.md

```diff
@@ -1,16 +1,4 @@
-# Base9 Documentation 
 
-## Introduction to Base9
-
-Base9 is a tiny virtual machine and programming language! We’re using it to show people how to use [Eclipse OMR] to build their own programming language. 
-
-[Eclipse OMR]: https://github.com/eclipse/omr
-
-The Base9 language is a subset of JavaScript and is very limited in functionality. It currently only contains support for integers. 
-
-For a full overview of Base9 and it's components, plus a tutorial on adding new bytecodes, see our [Supercharge a Language Runtime] presentation and tutorial. 
-
-[Supercharge a Language Runtime]: ./CASCON2017_SuperchargeALanguageRuntime.pdf
```

Comment:
JW why you removed this?

Reply:
I moved it to the index--It looked more like an introduction, so I thought it'd be a good starting point for our homepage.